bin/regression-wally

```diff
@@ -301,7 +301,8 @@ def addLockstepTestsByDir(dir, config, sim, fcovMode):
         for dirpath, dirnames, filenames in os.walk(os.path.abspath(dir)):
             for file in filenames:
                 # fcov lockstep only runs on WALLY-COV-ALL.elf files; other lockstep runs on all files
-                if (file.endswith(".elf") and fcovMode == 0 or file.endswith("ALL.elf") and fcovMode == 1):
+                if ((file.endswith(".elf") and (fcovMode == 0 or "tests/priv" in dir)) or 
```

Comment:
I know UET want to use some of the riscv-arch-tests for functional coverage. If you’re changing this anyway does it make sense to generalize so any of the riscof tests can be run with fcov instead of just priv tests?